src/main/java/seedu/mindmymoney/userfinancial/Expenditure.java

```diff
@@ -0,0 +1,35 @@
+package seedu.mindmymoney.userfinancial;
+
+/**
+ * Represents user's expenditure.
+ */
+public class Expenditure {
+    private String description;
+    private int amount;
+
+    public Expenditure(String description, int amount) {
+        setDescription(description);
+        setAmount(amount);
```

Comment:
Is it better if we do it this way? Then we don't need the setDescription and setAmount methods.
```suggestion
        this.description = description;
        this.amount = amount;
```